synapse/models/risk.py

```diff
@@ -168,7 +175,60 @@ def getModelDefs(self):
                         'doc': 'A confirmed attack that this alert indicates.',
                     }),
                 )),
-
+                ('risk:compromise', {}, (
+                    ('name', ('str', {'lower': True, 'onespace': True, 'strip': True}), {
+                        'doc': 'A brief name for the compromise event.',
+                    }),
+                    ('desc', ('str', {}), {
+                        'disp': {'hint': 'text'},
+                        'doc': 'A prose description of the compromise event.',
+                    }),
+                    ('type', ('risk:compromisetype', {}), {
+                        'ex': 'cno.breach',
+                        'doc': 'The compromise type.',
+                    }),
+                    ('target', ('ps:contact', {}), {
+                        'doc': 'Contact information of the target.',
+                    }),
+                    ('attacker', ('ps:contact', {}), {
+                        'doc': 'Contact information of the attacker.',
+                    }),
+                    ('campaign', ('ou:campaign', {}), {
+                        'doc': 'The campaign that this compromise is part of.',
+                    }),
+                    ('time', ('time', {}), {
+                        'doc': 'Earliest known evidence of compromise.',
+                    }),
+                    ('duration', ('duration', {}), {
+                        'doc': 'The duration of the compromise.',
+                    }),
+                    ('loss:pii', ('int', {}), {
+                        'doc': 'The number of records compromised which contain PII.',
+                    }),
+                    ('loss:econ', ('econ:price', {}), {
```

Comment:
:loss:* and :ransom:* feel like impacts to me, and therefore candidates for a separate form altogether.